middleware.js

```diff
@@ -1,11 +1,11 @@
 const prometheusClient = require("prom-client");
-const gcStats = require("prometheus-gc-stats");
+// const gcStats = require("prometheus-gc-stats");
 
 const prometheusRegistry = new prometheusClient.Registry();
 prometheusClient.collectDefaultMetrics({ register: prometheusRegistry });
 
-const startGcStats = gcStats(prometheusRegistry); // gcStats() would have the same effect in this case
-startGcStats();
+// const startGcStats = gcStats(prometheusRegistry); // gcStats() would have the same effect in this case
+// startGcStats();
```

Comment:
Why are these commented out?

Reply:
With `prometheus-gc-stats` dependency there are some vulnerabilities seen with one of it's dependency `tar` which is why we have currently commented it out!